src/main.js

```diff
@@ -230,16 +230,16 @@ const CodingManagerApplication = new Lang.Class({
         // way that ends up calling to hide() between the user pressed
```

Comment:
Looks good and works reliable here to hide and show the game manager.

I guess we can remove the comment then?